apps/admin/pages/api/auth/[...nextauth].ts

```diff
@@ -50,8 +50,7 @@ export default async function auth(req: any, res: any) {
           return Promise.resolve(true);
         }
 
-        if (account.provider === "google" && profile.verified_email === true)
-          return Promise.resolve(false);
+        if (account?.provider === "google") return Promise.resolve(false);
```

Comment:
Did google change?

Reply:
no, but verified_email isn't in the type and not needed.